src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -30,6 +30,105 @@ def tearDown(self):
     pass
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_4(self):
+    """Send Spectrum Inquiry from inside coastal exclusion zone.
+       With no ESC deployed, use a Category A device to request Spectrum
+       Inquiry.
+
+    The response should be Success, with NO channels in result.
+    """
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    # Locate this device close to coast (Half Moon Bay, Calif.)
+    device_a['installationParam']['latitude'] = 37.444267
+    device_a['installationParam']['longitude'] = -122.428628
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    cbsd_id = response['cbsdId']
```

Comment:
Maybe a check for Registration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`